### ISSUE TITLE
Explicitly binding capabilities triggers spurious warning 

### ISSUE THREAD

**Issue by b-studios** [effekt-lang/effekt#145]
```effekt
effect Exc(): Unit
def main() =
  try { e.Exc() } with e: Exc { () }
```
![image](https://user-images.githubusercontent.com/408265/191777056-43964eda-471d-4ef0-beb0-2bb11507e6e4.png)

Expected behavior: No warning.

**Comment by jiribenes** [comment]
Today I've independently ran into this issue as well.
<img width="363" alt="Screenshot 2022-09-23 at 10 36 51" src="https://user-images.githubusercontent.com/11269173/191922502-6d43d0f3-58c4-4320-8087-6af3beb013f1.png">

**Comment by b-studios** [comment]
Yes, this is guaranteed to occur every single time, since we do not consider bound capabilities as "effect usage" at the moment. :)
